packages/reports/reports.hjson

```diff
@@ -0,0 +1,212 @@
+{
+  "reports": [
+    {
+    "id": "8",
+    "config": {
+      "fetch": [
+        {
+          "fetchAnalytics": {
+            "dataElementCodes": [
+              "BCD46",
+              "BCD47",
+              "BCD48",
+              "BCD48a",
+              "BCD48b",
+              "BCD49",
+              "BCD50",
+              "BCD51",
+              "BCD52",
+              "BCD53",
+              "BCD54",
+              "BCD55"
+            ],
+            "aggregationType": "RAW",
+            "dataServices": [
+              {
+                "isDataRegional": true
+              }
+            ]
+          }
+        }
+      ],
+      "transform": [
+        {
+          "transform": "select",
+          "$.dataElement": "$.value",
+          "...": ["period", "organisationUnit"]
+        },
+        {
+          "transform": "sort",
+          "by": "$.period",
+        },
+        {
+          "transform": "aggregate",
+          "organisationUnit": "group", 
+          "...": "last"
+        },
+        {
+          "transform": "aggregate",
+          "organisationUnit": "drop",
+          "period": "drop",
+          "...": "sum"
+        },
```

Comment:
This is really powerful, and I love it, but also the sort of thing we'll probably pack together for the sake of data stewards into a saved block

Reply:
Yep I totally agree. It's basically just an existing aggregation type, so it could be something like:
```
{
          "transform": "finalValuePerPeriod",
          "period": "YEAR"
}
```
or even just:
```
 "finalValuePerYear"
```